### PR TITLE
Improve design for internal pages 2

### LINKED ISSUE #155: Improve internal page design (users, people, rooms, buildings, locations)
Internal pages should follow the general style guidelines

### Acceptance

- [x] /users/passwords/new matches the general style guidelines
- [x] /users/passwords/edit matches the general style guidelines
- [x] /users/sessions/new matches the general style guidelines
- [x] /users/users/sign_up matches the general style guidelines
- [x] /users/edit matches the general style guidelines
- [x] /people matches the general style guidelines
- [x] /people/n matches the general style guidelines
- [x] /people/n/edit matches the general style guidelines
- [x] /rooms matches the general style guidelines
- [x] /rooms/n matches the general style guidelines
- [x] /rooms/n/edit matches the general style guidelines
- [x] /buildings matches the general style guidelines
- [x] /buildings/n matches the general style guidelines
- [x] /buildings/n/edit matches the general style guidelines
- [x] /locations matches the general style guidelines
- [x] /locations/n matches the general style guidelines
- [x] /locations/n/edit matches the general style guidelines

### DoD

- [x] Tests
- [x] Documentation
- [x] CI Build passing
- [ ] Peer Review
- [ ] Approval from PO

### PR DESCRIPTION
**Pull request by taminob**
closes #155 (second part)

### CODE REVIEW

File: app/views/rooms/index.html.erb [hunk 1]
```diff
@@ -13,56 +13,12 @@
     </thead>
 
     <tbody>
+      <% Room.room_type_to_external_mapping.each do |internal,external| %>
```
Comment (bewee) [inline app/views/rooms/index.html.erb:16]: Nice!

**Comment by BenediktV** [comment]
There will be conflicts with other branches.
There are conflicts regarding the styling of the show pages with #131.
Also there is at least one issue with #127 because the back and edit buttons on the show pages should not get displayed anymore.

**Review by cambryx — CHANGES_REQUESTED** [review]
Looks really good!
We are a big fan of the rooms index page refactoring.

For some lines in tests we don't understand why they were removed.

Dev should be rebased into this branch before we can merge

File: spec/views/rooms/show.html.erb_spec.rb [hunk 2]
```diff
@@ -17,7 +17,6 @@
     render
     expect(rendered).to match(/Name/)
     expect(rendered).to match(/Floor/)
-    expect(rendered).to match(/Room Type/)
```
[the remark this answers is not in the document]
Comment (bewee) [reply spec/views/rooms/show.html.erb_spec.rb:20]: ups

File: spec/views/rooms/edit.html.erb_spec.rb [hunk 3]
```diff
@@ -21,8 +21,6 @@
       assert_select "input[name=?]", "room[name]"
 
       assert_select "input[name=?]", "room[floor]"
-
-      assert_select "input[name=?]", "room[room_type]"
```
[the remark this answers is not in the document]
Comment (bewee) [reply spec/views/rooms/edit.html.erb_spec.rb:25]: definitely

**Review by cambryx — APPROVED** [review]
Looks good

**Review by Tratori — APPROVED** [review]

**Pull request merged**